backend/graminsta/post/models.py

```diff
@@ -12,9 +13,13 @@ class Post(models.Model):
     publisher = models.ForeignKey(auth_models.User, on_delete=models.CASCADE,
                                   related_name="published_posts")
     description = models.TextField()
-    img = models.ImageField(upload_to='img')
+    lng = models.FloatField(default=100)
```

Comment:
I think you don't need default value. Maybe `blank=True` and `null=True` can help?

```
lon = models.FloatField(blank=True, null=True)
lat = models.FloatField(blank=True, null=True)
```